proposals/0064-remote-control.md

```diff
@@ -0,0 +1,812 @@
+  <function name="OnReverseAppsAllowing" messagetype="notification">
+    <description>Sender: vehicle -> RSDL. Notification about remote-control for passenger`s applications must be either turned off or turned on. Sent after User`s choice through HMI.</description>
+    <param name="allowed" type="Boolean" mandatory="true" >
+      <description>If "true" - RC for passengers is allowed; if "false" - disallowed.</description>
+    </param>
+  </function>
+  <function name="OnDeviceRankChanged" messagetype="notification">
+    <description>Sender: vehicle->RSDL. Purpose: inform about the device is set either as driver`s or as passenger`s  device by the HMI settings or by the HMI User through the appropriate menu.</description>
+    <param name="deviceRank" type="Common.DeviceRank" mandatory="true">
+      <description>If "DRIVER" - the named by "DeviceInfo" device is set as driver`s. If "PASSENGER" -  the named by "DeviceInfo" device is set as passenger`s. </description>
+    </param>
+    <param name="device" type="Common.DeviceInfo" mandatory="true">
+      <description>The device info: name and ID. Initially is sent by SDL via UpdateDeviceList to the vehicle.</description>
+    </param>
+  </function>
+  <function name="OnDeviceLocationChanged" messagetype="notification">
+    <description>Sender: vehicle->RSDL. Purpose: inform about the device location in the vehicle interior.</description>
+    <param name="deviceLocation" type="Common.InteriorZone" mandatory="true">
+      <description>Defines the vehicle zone where the named device is located. </description>
+    </param>
+    <param name="device" type="Common.DeviceInfo" mandatory="true">
+      <description>The device info: name and ID. Initially is sent by SDL via UpdateDeviceList to HMI.</description>
+    </param>
+  </function>
+</interface>
+```
+
+## Potential downsides
+
+- The driver distraction rules and regional regulations are getting more stringent on driver's using mobile phones while driving. Current design cannot prevent a driver mark a phone as PASSENGER and use it while driving.
+
+- The driver must exit current control application before using another application to control the same RC module. There is no indication of which application is currently control the RC module. Driver doesn't know which application to be closed. This makes application switching cumbersome.
+
+- It lacks the policy control on which application can access which RC resource.
+
+- RPC messages are not encrypted. Attackers may try to eves-drop and spoof wireless communication.
+
+- It can only subscribe to all radio or climate control data, cannot subscribe to individual item change notifications.
+
+- RC capabilities does not include detailed control item, it contains only moduleDescription (zone + type).
+
+- DeviceRank is in OnHMIStatus notification, preferred to be in a new separate notification.
+
+
+## Impact on existing code
+
+Code is in SDL-RC feature branch and ready to merge.
```

Comment:
This is a misleading sentence. Code that should reflect these features are in GitHub for Core, however, it hasn't been reviewed at all and can't be stated as "ready to merge". Also, each of the proxies libraries implementations are likely not complete and are not in a merge ready state either.